File: Library_python /Media.py
class Media(object):
   def __init__(self, call_no, title, subject):
      self.callNumber = call_no
      self.title = title
      self.subject = subject

   def display(self):
      print ("Call Number: ", self.callNumber, "Title: ", self.title, "Subject: ", self.subject)
   
   def comp_call_no(self, call_no):
      if call_no in self.callNumber:
         return True
      else:
         return False

   def comp_title(self, title):
      if title in self.title:
         return True
      else:
         return False

   def comp_subjects(self, subject):
      if subject in self.subject:
         return True
      else:
         return False

File: Library/media.cpp
#include<iostream>
#include"media.h"

Media::Media(const string& call_no, const string& title, const string& subject,const  string& notes): m_call_no(call_no),m_title (title),m_subjects (subject),m_notes (notes)
{
}

void Media:: display() const
{
    cout<<"Call_number: "<<m_call_no;
    cout<<"Title: "<<m_title;
    cout<<"Subjects: "<<m_subjects;
    cout<<"Notes: "<<m_notes;
}

bool Media::comp_call_no(const string& str) const
{
    if(m_call_no.find(str) != string::npos)
    {
        return true;
    }
    else
    {
        return false;
    }
}
bool Media::comp_title(const string& str) const
{
    if(m_title.find(str) != string::npos)
    {
        return true;
    }
    else
    {
        return false;
    }
}
bool Media::comp_subjects(const string& str) const
{
    if(m_subjects.find(str) != string::npos)
    {
        return true;
    }
    else
    {
        return false;
    }
}

Media :: ~Media() //Virtual Media destructor
{
}
